easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.15-foss-2018a-Perl-5.26.1.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.15'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'optarch': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['31e47afd5be679d89ece811a227525925b6907cce4af2c86f10f465e080383e3']
+
+dependencies = [
+    ('Perl', '5.26.1'),
```

Comment:
style nitpicking: this should be in a single line

Reply:
@migueldiascosta Actually, we're not consistent there at all, and as long as the style check run by Travis doesn't complain about this sort of thing, we shouldn't block PRs over this (granted, you could say the same about SHA256 checksums, but we should really get that fixed ;-)).